src/databricks/labs/ucx/source_code/files.py

```diff
@@ -39,6 +40,11 @@ def build_dependency_graph(self, parent: DependencyGraph) -> list[DependencyProb
     def path(self) -> Path:
         return self._path
 
+    def __eq__(self, other: Any) -> bool:
```

Comment:
Maybe remove this

Reply:
It's needed for the entity to be used as dictionary key